### DISCUSSION TITLE
Custom voices

### DISCUSSION THREAD

**Discussion by POD-io** [ILikeAI/AlwaysReddy#57] [Ideas]
I started looking into training a model for Piper, which seems possible. But I remembered another system I have used called alltalk_tts which is insanely easy to clone a voice. You simply drop a .wav file up to 30 seconds in a folder and it generates a clone for you.

Anyone experiment with custom voices and this project? The Piper training would probably work but seemed a bit heavy to do.

**Comment by ILikeAI** [comment]
I have heard of all talk but haven't had a chance to play with it yet. There is someone in the discord who has successfully trained piper and may be able to help you out if this is something your are interested in

**Comment by ILikeAI** [comment]
I have heard of all talk but haven't had a chance to play with it yet. There is someone in the discord who has successfully trained piper and may be able to help you out if this is something your are interested in

**Reply by POD-io** [reply]
> I have heard of all talk but haven't had a chance to play with it yet. There is someone in the discord who has successfully trained piper and may be able to help you out if this is something your are interested in

Oooh, I did tinker with the Piper training but it seemed more than I wanted to dive into.
I was able to successfully get alltalk_tts configured, which involves just dropping any 30s clip of a voice into a folder. Now, AlwaysReddy takes on that voice when speaking. All running locally.

That said, I would wager the Piper implementation/training is much better. I haven't tried too many voices yet with alltalk, but it works well enough.
